lib/internal/bootstrap_node.js

```diff
@@ -462,7 +462,13 @@
     }
 
     if (!NativeModule.exists(id)) {
-      throw new Error(`No such native module ${id}`);
+      // Model the error off the internal/errors.js model, but
+      // do not use that module given that it could actually be
+      // the one causing the error if there's a bug in Node.js
+      const err = new Error(`No such built-in module: ${id}`);
+      err.code = 'ERR_UNKNOWN_BUILTIN_MODULE';
+      err.name = 'Error[ERR_UNKNOWN_BUILTIN_MODULE]';
```

Comment:
nit: there should be a space now that https://github.com/nodejs/node/commit/7b4a72d797cc57bc69aa6be22c34505a456f7b3a has landed.